### ISSUE TITLE
Improvement: Display a warning message when the Sucuri API has a bad format

### ISSUE THREAD

**Issue by WordPresseur** [wp-media/wp-rocket#1356]
If you enter a wrong format API in the Sucuri Add-on and you save the settings, it just doesn't keep the API in the field. No alert message at all. This can be confusing for the customer.

Ref: https://wp-media.slack.com/archives/C08N8J6VC/p1542208011107400

**Comment by webtrainingwheels** [comment]
I do get the admin notice as expected when entering an incorrect key. 👍

But there's a couple of quirks: 
1. I got the admin notice immediately after updating a site where the add-on wasn't enabled: 
![wp_rocket_ _wordpress_themes_ _wordpress](https://user-images.githubusercontent.com/1043076/49622922-60cd5680-f980-11e8-91cd-56443118863f.jpg)
2. Similarly, on a site where the add-on was active and I input the wrong key, I got the notice as expected. But after turning off the add-on, the notice did not go away.
